cmd/handoff-bss-metadata.go

```diff
@@ -60,6 +61,14 @@ var handoffBSSMetadataCmd = &cobra.Command{
 	Short: "runs migration steps to build BSS entries for all NCNs",
 	Long:  "Using PIT configuration builds kernel command line arguments and cloud-init metadata for each NCN",
 	Run: func(cmd *cobra.Command, args []string) {
+
+		// 1.x uses vlan names, 1.2.x uses new names
+		if csmVer == "1.0" || csmVer == "1.0.1" {
```

Comment:
Will they have this in both contexts? I thought we shipped a different version for 1.0.x and 1.2.

Reply:
This RPM and subsequent changes will ship with 1.2.  But in order to facilitate upgrades, the users will have to install a newer version of `csi` then what was shipped with 1.0.  

Thus, this only applies for upgrades and is called manually with an additional flag on the command line.  Otherwise, it behaves as it currently does.